pytidb/embeddings/builtin.py

```diff
@@ -203,7 +221,11 @@ def _process_image_source(self, source: SourceInputType) -> Union[str, dict]:
             else:
                 raise ValueError(f"invalid url format for image source: {source}")
         elif isinstance(source, Image):
-            return {"image": encode_pil_image_to_base64(source)}
+            base64_str = encode_pil_image_to_base64(source, compress_for_bedrock=self.model_name.startswith("bedrock/"))
+            # For bedrock models, return base64 string directly
+            if self.model_name.startswith("bedrock/"):
```

Comment:
`data:[<media-type>][;base64],<data>` has a name called [Data URLs](https://developer.mozilla.org/en-US/docs/Web/URI/Reference/Schemes/data), Strictly speaking, base64 string is just a part of it.

It's better to move the format process parts from `encode_pil_image_to_base64` to `_process_image_source` function.

For example:

```suggestion
            base64_str = encode_pil_image_to_base64(....)
            if self.model_name.startswith("bedrock/"):
                return f"data:image/jpeg;base64,{base64_str}"
            else:
                return {"image": base64_str}
```